### DISCUSSION TITLE
[source-postgresql, cdc] On append ingestion issue data are lost

### DISCUSSION THREAD

**Discussion by ftapin** [airbytehq/airbyte#33720] [Connector Ideas and Features]
Hi there,

We found a strange issue on Airbyte

### **Context**
We are actually ingesting PostgreSQL CDC data through Airbyte to Biguery table.
When an incremental ingestion failed we lost CDC data.
![Capture d’écran 2023-12-21 à 17 24 30](https://github.com/airbytehq/airbyte/assets/100791306/a5dfc264-eb4d-40c2-99ff-bde1192bf29f)

Ingestion is written as successfull but data are not loaded into BigQuery table.

Do you have any idea to solve this issue? 

### **Technical informations**

| Connector Name  | Version  | Link |
| :---: | :---: | :---: |
| Postgres | 3.3.0 | [airbyte/source-postgres](https://docs.airbyte.com/integrations/sources/postgres)|

**Comment by marcosmarxm** [comment]
What is the **LSN commit behaviour** parameter you're using? (Please check the connector setting page)

[a comment by an automated account is not shown]
